src/ErrorHandler/IErrorHandler.php

```diff
@@ -0,0 +1,19 @@
+<?php declare(strict_types = 1);
+
+namespace Apitte\Core\ErrorHandler;
+
+use Throwable;
+
+interface IErrorHandler
+{
+
+	public function setCatchException(bool $catchException): void;
```

Comment:
Tohle je vec implementace jednotliveho ErrorHandleru. Neni to potreba definovat v interface.